sorc/chgres_cube.fd/input_data.F90

```diff
@@ -172,126 +172,126 @@ subroutine read_input_nst_data(localpet)
  c_d_input_grid = ESMF_FieldCreate(input_grid, &
                                    typekind=ESMF_TYPEKIND_R8, &
                                    staggerloc=ESMF_STAGGERLOC_CENTER, rc=rc)
- if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
```

Comment:
@jedwards4b @GeorgeGayno-NOAA I see Jim removed those changes from the PR to ufs_release_v1.0. This is good, because the code that was reintroduced by commit 4b91f3380266a3d3d27ce7d753441656761917cb is not correct and incompatible with the GNU compilers. CPP directives have to be uppercase, i.e.
```
if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__))
```
is correct but
```
if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__))
```
is not. @GeorgeGayno-NOAA can you please revert this change for the develop branch?